docs/Software Architecture Document/Components View/index.md

```diff
@@ -214,12 +242,12 @@ The view is represented by module and subsystem diagrams that show the system's
     - Implements specific mobile RPC handling.
     - Implements specific HMI RPC handling.
   - *Relations*
-    - Composed by ***Plugin manager*** 
-    - Handles ***Application Manager*** by ***Service*** interface
+    - Composed by ***Plugin*** 
+    - Handles ***Application Manager*** by ***Application Manager*** interface
   - *Interfaces*
-    - Provides ***Plugin Manager*** interface   
+    - Provides ***Plugin*** interface   
   - *Behavior*
-    - Receives data from CoreService
+    - Receives data from ApplicationManager
     - Parses data
     - Creates commands.
     - Handles incoming HMI notifications
```

Comment:
aslo 
sdl_core_documentation/docs/Software Architecture Document/View-to-View Relations/index.md